test/terra/backends/aer_simulator/test_options.py

```diff
@@ -0,0 +1,92 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+AerSimualtor options tests
+"""
+
+from ddt import ddt
+from qiskit import QuantumCircuit, transpile
+from .aer_simulator_test_case import (
+    AerSimulatorTestCase, supported_methods)
+
+
+@ddt
+class TestOptions(AerSimulatorTestCase):
```

Comment:
Do we have variants of this test for `set_options` and kwargs on `run()`?

Reply:
I would just add those as separate test functions. I added class and run tests for seed_simulator, and we can add more to it later.